platform/platform_mac.mm

```diff
@@ -72,6 +81,9 @@
     }
   }
 
+  m_resourcesDir += '/';
```

Comment:
@kshalnev done
@ygorshenin Yes,
1. It is what this file does in other cases.
2. Otherwise we cant errors of the kind "cannot open omim/datasettings.ini".